kadai-adapter/src/main/java/io/kadai/adapter/configuration/AdapterConfiguration.java

```diff
@@ -40,42 +39,34 @@
 @EnableTransactionManagement
 public class AdapterConfiguration {
 
-  @Value("${.jndi-name:no-jndi-configured}")
-  private String jndiName;
-
-  @Bean
-  @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
-  public AdapterManager manager() {
-    return new AdapterManager();
-  }
-
   @Bean
   @Scope(ConfigurableBeanFactory.SCOPE_SINGLETON)
-  public ReferencedTaskCompleter referencedTaskCompleter() {
-    return new ReferencedTaskCompleter();
+  public ReferencedTaskCompleter referencedTaskCompleter(final AdapterManager adapterManager) {
```

Comment:
Why did you add new parameters

Reply:
For constructor injection instead of `@Autowired`